src/js/modules/infragistics.ui.editors.js

```diff
@@ -10664,8 +10672,8 @@
 					} else {
 
 						//T.P. 10th Dec 2015 211062: When there is no value and the datepicker selects value the stored date object needs to be with current time.
-						//In Case there is no dateObject which meand the editor has no value when the date is selected it will be with current time value (hours, minutes, seconds)
-						date = new Date();
+						//In Case there is no dateObject, which means the editor has no value when the date is selected it will be with current time value (hours, minutes, seconds)
```

Comment:
Can you check with the issue in the comment why this was added. You could probably remove the entire else branch, since the jQuery datepicker will return the selected date with 0 as hours/min/sec.